### PR TITLE
Post data( add courses)

### PR DESCRIPTION
**Pull request by RandInaim**
we added a form to add courses and all its needed functionalities

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -47,4 +50,20 @@ app.get('/course/:id', (req, res, next) => {
...
 })
 
+
+app.post('/addCourse',(req,res,next)=>{
+    const {name,description,costs,teacherName,available,courseTime,img}=req.body;
+    const courseData = { name, description, costs, teacherName, available, courseTime, img }
+   
+    
+    postData.addCourse(courseData)
+        .then(()=>{
```
Comment (ghassanmas) [inline src/app.js]: why you don't say `postData.addCourse(req.body)`?

File: src/app.js [hunk 2]
```diff
@@ -24,8 +24,11 @@ app.engine(
 )
 
 app.get('/', (req, res, next) => {
+    
     getData.getcourses()
         .then((course) => {
+
+        
             res.render('home', {
```
Comment (ghassanmas) [inline src/app.js]: Why these unneeded spaces? run eslint before pushing

File: src/app.js (hunk 1 above)
Comment (ghassanmas) [reply src/app.js]: or just say `const course = req.body`

Comment (RandInaim) [reply src/app.js]: fixed with the first choice

**Pull request merged**